### ISSUE TITLE
Terminal is significantly scaled down on MacOS

### ISSUE THREAD

**Issue by l4tte** [cfyzium/bearlibterminal#3]
The code
```C++
#include <stdio.h>
#include "BearLibTerminal.h"

int main()
{
    terminal_open();

    terminal_print(1, 1, "Hello, world!");
    terminal_refresh();

    while (terminal_read() != TK_CLOSE);

    terminal_close();
}
```

As taken from [here](http://foo.wyrd.name/en:bearlibterminal) will compile and run just fine, but the text is significantly scaled to 25% of the display

![image](https://user-images.githubusercontent.com/34426037/94749053-dd4cd880-0350-11eb-9389-53010492d1ad.png)

This happens with all BearLibTerminal terminals, for example, the SampleOmni program looks like this

![image](https://user-images.githubusercontent.com/34426037/94749066-e5a51380-0350-11eb-828e-e7fbff866fac.png)

And the mouse program looks like this (Mouse is in top right corner of screen)

![image](https://user-images.githubusercontent.com/34426037/94749087-edfd4e80-0350-11eb-85c7-8122bb2c6d4e.png)

**Comment by redblobgames** [comment]
My guess is that there's a mismatch between the window size in "points" and "pixels", and bearlibterminal is using one when it needs to use the other. I don't know how to fix it though.

**Comment by l4tte** [comment]
I've been messing around with it, and you're right. It's an issue with glut too for the same reason

**Comment by quulah** [comment]
Has someone managed to workaround this or find the root cause?

**Comment by hazeled** [comment]
It's most likely because of MacOS' retina HiDPI screens, they're scaled up and have more pixels than a standard screen

**Comment by quulah** [comment]
Yeah, sounds likely. I'm guessing we're waiting for this then: #9
